src/components/FeaturedProjectCard.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import styled from 'styled-components';
+import FeaturedProject from 'FeaturedProject.json';
+
+const FeaturedProjectCard = ({ color, tagColor }) => {
+  const GITHUB_NAME = 'Sofia-Andersson';
+
+  return (
+    <Wrapper>
+      <SectionHeader color={color}>FEATURED PROJECTS</SectionHeader>
+      <FeaturedProjectWrapper>
+        {FeaturedProject.map((project) => {
+          return (
+            <ProjectCard
+              key={project.title}
+              href={`https://github.com/${GITHUB_NAME}/${project['repo-name']}`}
+              target="_blank">
+              <ThumbnailWrapper url={project.image}>
+                <ThumbnailTitle>{project.title.toUpperCase()}</ThumbnailTitle>
+              </ThumbnailWrapper>
+              <ProjectInfoHeader color={color}>
+                {project['title-description'].toUpperCase()}
+              </ProjectInfoHeader>
+              <ProjectInfo>{project['project-description']}</ProjectInfo>
+              <ul style={{ padding: 0 }}>
+                {project.tags.map((tag, i) => (
+                  // eslint-disable-next-line react/no-array-index-key
+                  <ProjectTags tagColor={tagColor} key={i}>
+                    {tag}
+                  </ProjectTags>
+                ))}
+              </ul>
+            </ProjectCard>
+          );
+        })}
+      </FeaturedProjectWrapper>
+    </Wrapper>
+  );
+};
```

Comment:
very nice to map throu your json saves alot of time, and hassel. i have a similar  since our AWESOME team help eachother !